### PR TITLE
Add faq.md in Spanish language

### PR DESCRIPTION
**Pull request by miporto**
Hi! I have translated and reviewed the whole faq file to Spanish. It still may have some errors though.

### CODE REVIEW

**Comment by durden** [comment]
Thanks for the translation!  We're still working on the best way to manage translations and serve up the right version or create links on the site for different languages.  I'll make sure to post back here with some ideas and feel free to let us know if you have any!

**Comment by miporto** [comment]
That's ok, I'll try to mantain the translation updated if you're ok with it. Also, if I come up with an idea I'll make sure to post it here!

File: faq_spanish.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+¡Bienvenidos! Esta es una comunidad para colaborar en la creación de guías técnicas y tutoriales. Nuestro visión colectiva es que, trabajando juntos podemos educar al mundo, y además mejorar nuestro conocimiento. Piensen de esto como una "Wikipedia" de guías técnicas y tutoriales, o  como "Medium.com para desarrolladores".
```
Comment (elbrujohalcon) [inline faq_spanish.md:1]: Nuestro visión -> Nuestra visión
Piensen de esto -> Piensen en esto

(Spanish speaker from Buenos Aires, here)

File: faq_spanish.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+- No existe una manera, para los desarrolladores, de mejorar el contenido mediante revisión de pares. WordPress no es bueno para esto.
+- Actualmente las guías existen en silos. Viven en blogs individuales, están fragmentadas y varían en cuanto a calidad. Típicamente, estas guías no alcanzan a una gran audiencia ya que los blogs individuales tienen un alcance limitado.
+- Las guías sobre programación están mayormente en Inglés y no se traducen ni rápida ni fácilmente a otros idiomas ya que el código no es fácilmente traducible. Otros países no tienen un acceso sencillo a la educación.
+
+### Cómo estamos resolviendo estos problemas
+
+- **Usamos [MarkDown](http://daringfireball.net/projects/markdown/basics)** (en vez de WordPress), un simple formato de archivos. Es un formato de archivos de texto, extremadamente sencillo, que toma alrededor de 60 segundos en entenderse. Es realmente sencillo hacer que el texto esté en **negrita**, _ itálica _, `bloque de código`, etc. MarkDown es utilizado en muchos lugares, como los archivos README de Github. Incluso este archivo está escrito en MarkDown, tu puedes ayudarnos a mejorarlo.
```
Comment (elbrujohalcon) [inline faq_spanish.md:13]: tu puedes -> tú puedes

File: faq_spanish.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+- **Las guías se almacenan en Github**, que muchos desarrolladores aman y utilizan diariamente.
+- **Usa tu editor de texto favorito** para crear tutoriales y **revisarlos vía Git**. O también puedes editarlos utilizando la interfaz de usuario de Github. **Mucho mejor que WordPress**.
+- Si no estás familiarizado con git/Github, o no quieres molestarte con eso, está bien -- tenemos un conveniente editor en nuestro sitio para editar tutoriales justo aquí.
+- **El código de este sitio es de código abierto y está en Github**. ¡Mejorémoslo juntos!
+- Puedes tener tus tutoriales **revisados por otros**. Esto mejora la calidad de nuestro conocimiento colectivo.
+- **Otros pueden ayudar a mantener tus tutoriales en el largo plazo**, logrando que el código tenga una mayor vigencia cuando actualizaciones son necesarias, y te permite transferir la autoría si estás muy ocupado y no puedes mantenerlo.
```
Comment (elbrujohalcon) [inline faq_spanish.md:19]: actualizaciones son necesarias -> se requieran actualizaciones
y te permite -> y permitiendo

File: faq_spanish.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+- **Podemos ayudarnos entre nosotros a alcanzar a una audiencia más amplia y generar un mayor impacto**. Cuando un tutorial es publicado, todos trabajamos juntos para promocionarlo en HackerNews, y así atraer muchos lectores. Impulsando este sitio, juntos obtendremos una mayor audiencia con el correr del tiempo, lo que significa que tus tutoriales llegarán a una mayor audiencia.
+- (Futuro) **Llevar conocimiento a otros países**. Tu contenido puede ser traducido por una comunidad  bilingüe y apasionada para que sus países reciban conocimiento, para que sea compartido con otros países que no tienen un gran conocimiento del idioma Inglés.
```
Comment (elbrujohalcon) [inline faq_spanish.md:21]: conocimiento del idioma -> dominio del idioma

(the idea here is not to use "conocimiento" twice in the same sentence)

File: faq_spanish.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+* **Conéctate con otros importantes miembros de esta comunidad**. Te daremos acceso a nuestra comunidad en [Slack](https://hackguides.typeform.com/to/ZroxPD), para conectarte con otros asombrosos miembros que también estén involucrados.
+* **Acceso gratuito para la próxima conferencia hack.summit() para ti y tus amigos**. ¡Obtén tu código especial para poder compartirlo!
+
+### ¿Quién es el dueño de los tutoriales creados aquí?
+Tu lo eres, al ser el autor original. Este sitio es un lugar para que tu contenido sea mantenido en una forma mucho mejor que en WordPress, y que alcance a una audiencia más amplia. Pero no somos dueños de tus obras. De hecho, si quisieras, podrias hostear una copia de este sitio, ya que todo aquí es código abierto.
+
+### ¿Qué tipos de tutoriales son apropiados para publicar?
+*Siéntete libre para escribir sobre cualquier tecnología sobre la que eres apasionado.*  En líneas generales, alguno de los siguientes:
+
+* Tutorial sobre X (donde X es una tecnología o librería por la que eres apasionado).
```
Comment (elbrujohalcon) [inline faq_spanish.md:36]: por la que eres apasionado -> que te apasiona

File: faq_spanish.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+* 10 de cosas que probablemente no conoces de X
```
Comment (elbrujohalcon) [inline faq_spanish.md:37]: 10 de cosas -> 10 cosas

File: faq_spanish.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+* Ofrécete para ayudar a otros a mejorar su contenido.
+* Haz contribuciones al código de nuestro sitio.
+* Haz correr la voz sobre el sitio en Twitter, Facebook, HackerNews, etc.
+* Ofrécete a traducir para ayudarnos a brindar contenido a otros idiomas.
+
+Si quieres charlar sobre esto, por favor contáctate con Prateek (prateek-gupta@pluralsight.com) o háblanos por Slack!
+
+### ¿Cómo se si mi idea para un tutorial es buena?
+Algo sencillo que puedes hacer es revisar que existe sobre esa idea en particular. Prueba haciendo una búsqueda rápida sobre temas que te interesen. Otros articulos en general están bien, siempre y cuando haya suficiente contenido que establezca a tu artículo aparte. ¿Hay ideas sobre las que quieres hablar en tu artículo que aún no hayan sido escritas? ¿Suficientes contribuciones únicas que logren hacer avanzar al conocimiento mundial? Esas son algunas de las cosas a tener en cuenta.
```
Comment (elbrujohalcon) [inline faq_spanish.md:64]: que existe -> qué existe
que establezca a tu artículo aparte -> que distinga a tu artículo de los demás

File: faq_spanish.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+### ¿Cómo es el proceso de publicación?
+
+1. [Escribe un tutorial](/write/)
+2. Comparte un link para obtener sugerencias de tus amigos y colegas para tu artículo.
+3. Continúa mejorando tu artículo con la ayuda de la comunidad.
+4. Nuestros editores también te ayudarán para mejorar tu artículo.
```
Comment (elbrujohalcon) [inline faq_spanish.md:71]: para mejorar -> a mejorar

**Pull request closed without merging**